service/src/main/resources/config/verily/sdd/original/ui/top_level.json

```diff
@@ -183,7 +208,8 @@
         { "key": "t_rollup_count", "width": 120, "title": "Roll-up count" }
       ],
       "occurrence": "observation_occurrence",
-      "classification": "observation"
+      "classification": "observation",
+      "modifiers": ["age_at_occurrence", "visit_type", "date_group_by_count"]
```

Comment:
Does the `date_group_by_count` modifier refer to the `measurement_occurrence.date` attribute? And if so, why is it different from the other two with the `group_by_count` suffix?

Reply:
These refer to the `modifierConfigs` defined below. The difference between them is that `condition_occurrence` and `ingredient_occurrence` have `start_date` and `end_date` while the others just have a `date` so they are group by different attributes.